xml/1.3/modules/ROOT/pages/xml-schema-validation.adoc

```diff
@@ -26,15 +54,12 @@ By default, this operation looks for the input document at the message payload l
 </flow>
 ----
 
-Note that you can use the `<xml-module:validate-schema>` component inside a `<validation:all>` element.
-
-== Handling the Validation Error
 
-If the validation is successful, nothing happens, and the flow continues to the next operation. If it fails, an `XML-MODULE:SCHEMA_NOT_HONOURED` error is raised.
+== Manage the Validation Error
 
-The `XML-MODULE:SCHEMA_NOT_HONOURED` error is a complex one. Because the validation can fail for any number of reasons, the error contains a list of Messages. Each message contains a `SchemaViolation` object, which has the following structure:
+If the validation is successful, the flow continues to the next operation. However, if the validation fails, an `XML-MODULE:SCHEMA_NOT_HONOURED` error is raised. +
+Because the validation can fail for any number of reasons, the error description contains a list of messages. Each message contains a `SchemaViolation` object, that has the following structure:
```

Comment:
```suggestion
Because the validation can fail for more than one reason, the error description contains a list of messages. Each message contains a `SchemaViolation` object that has the following structure:
```